sklearn/ensemble/_voting.py

```diff
@@ -64,8 +64,12 @@ def fit(self, X, y, sample_weight=None):
 
         self.estimators_ = Parallel(n_jobs=self.n_jobs)(
                 delayed(_parallel_fit_estimator)(clone(clf), X, y,
```

Comment:
Could you put `clone(clf), ....` on the next line. It will give you some space to call `self._log_message()` on the same line).